<ui version="4.0" >
 <class>UPnPDialog</class>
 <widget class="QWidget" name="UPnPDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>UPnP Forwarder</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="routertabs" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1035</width>
        <height>418</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" />
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Re&amp;mote Host</string>
       </property>
       <property name="buddy" >
        <cstring>edit_remoteHost</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&amp;External Port</string>
       </property>
       <property name="buddy" >
        <cstring>edit_externalPort</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&amp;Protocol</string>
       </property>
       <property name="buddy" >
        <cstring>edit_Protocol</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>&amp;Internal Port</string>
       </property>
       <property name="buddy" >
        <cstring>edit_internalPort</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Internal &amp;Client</string>
       </property>
       <property name="buddy" >
        <cstring>edit_internalClient</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="5" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>E&amp;nabled</string>
       </property>
       <property name="buddy" >
        <cstring>checkBox_enabled</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="6" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>&amp;Description</string>
       </property>
       <property name="buddy" >
        <cstring>edit_description</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="7" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>&amp;Lease Duration</string>
       </property>
       <property name="buddy" >
        <cstring>edit_leaseDuration</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLineEdit" name="edit_remoteHost" />
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="edit_externalPort" />
     </item>
     <item row="1" column="2" >
      <widget class="QComboBox" name="edit_Protocol" >
       <property name="editable" >
        <bool>false</bool>
       </property>
       <item>
        <property name="text" >
         <string>TCP</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>UDP</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3" >
      <widget class="QLineEdit" name="edit_internalPort" />
     </item>
     <item row="1" column="4" >
      <widget class="QLineEdit" name="edit_internalClient" />
     </item>
     <item row="1" column="5" >
      <widget class="QCheckBox" name="checkBox_enabled" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="6" >
      <widget class="QLineEdit" name="edit_description" />
     </item>
     <item row="1" column="7" >
      <widget class="QLineEdit" name="edit_leaseDuration" />
     </item>
     <item row="1" column="8" >
      <widget class="QPushButton" name="button_newEntry" >
       <property name="text" >
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8" >
      <widget class="QPushButton" name="button_remove" >
       <property name="text" >
        <string>&amp;Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
